proof-systems/src/darlin/tests/mod.rs

```diff
@@ -457,6 +533,7 @@ mod test {
             &committer_key_g2,
             &verifier_key_g1,
             &verifier_key_g2,
+            None,
```

Comment:
Doing the same as previous tests also here won't harm

Reply:
Done